src/tests/multihost/alltests/test_ldap_extra_attrs.py

```diff
@@ -193,3 +196,60 @@ def test_0006_bz1667252(self, multihost):
         cmd = multihost.client[0].run_command(sssctl_cmd)
         ret = multihost.client[0].service_sssd('status')
         assert cmd.returncode == 0 and ret == 0
+
+    @staticmethod
+    @pytest.mark.tier1
+    def test_bz847043(multihost, backupsssdconf):
+        """
+        :title: Thread issue can cause the application to not get
+            any identity information bz847043
+        :id: a3f5b5ea-9cc6-11ed-98f2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=847043
+        :steps:
+            1. rm -f /var/lib/sss/mc/*
```

Comment:
This might be described as clear sssd caches and  it would fit more to the setup.